aten/src/ATen/native/quantized/cpu/qlinear.cpp

```diff
@@ -108,14 +107,34 @@ class QLinearInt8 final : public torch::OperatorKernel {
     fbgemm::DoNothing<> doNothingObj{};
 
     const int32_t* bias_ptr = nullptr;
-    if (bias.has_value()) {
-      Tensor bias_vec = bias.value();
-      TORCH_CHECK(bias_vec.dim() == 1, "bias should be a vector (1D Tensor)");
+    at::Tensor qbias;
+    if (pack_ptr.bias.has_value()) {
+      at::Tensor bias = pack_ptr.bias.value();
+      // Temporary: Quantize bias
+      qbias = bias;
+      if (pack_ptr.q_scheme == kPerTensorAffine) {
+        qbias = at::quantize_linear(
+            at::dequantize(bias),
+            pack_ptr.w_scale[0] * input_scale_float,
+            0,
+            kQInt32);
+      } else if (pack_ptr.q_scheme == kPerChannelAffine) {
+        std::vector<int64_t> vect(1, 0);
+        IntArrayRef axis(vect.data(), 1);
+        at::Tensor bias_scale = at::ones({N}, at::dtype(at::kDouble));
+        at::Tensor bias_zp = at::zeros({N}, at::dtype(at::kLong));
+        for (int i = 0; i < N; ++i) {
+          bias_scale.data_ptr<double>()[i] =
+              pack_ptr.w_scale[i] * input_scale_float;
+        }
+        qbias = quantize_linear_per_channel_cpu(
```

Comment:
Use quantize_linear_per_channel instead of quantize_linear_per_channel_cpu. quantize_linear_per_channel is the api exposed in Python frontend.